README.md

```diff
@@ -1 +1,25 @@
-# edge-api-utils
\ No newline at end of file
+# edge-api-utils
+
+Copyright (C) 2021 The Open Library Foundation
+
+This software is distributed under the terms of the Apache License,
+Version 2.0. See the file "[LICENSE](LICENSE)" for more information.
+
+## Introduction
+The purpose of this edge API utils is the combine classes/utilities/constants shared by both modules: edge-common and edge-common-spring.
+
+## Additional information
+
+### Required Permissions
+
+### Configuration
+Please refer to the [Configuration](https://github.com/folio-org/edge-common-spring/blob/master/README.md) section in the [edge-common](https://github.com/folio-org/edge-common-spring/blob/master/README.md) documentation to see all available system properties and their default values.
+
+### Issue tracker
+See project [EDGDEMATIC](https://issues.folio.org/browse/UXPROD-2973)
```

Comment:
Please reference the EDGAPIUTL JIRA project

Reply:
Changed to `https://issues.folio.org/browse/EDGAPIUTL-1`